vite.config.ts

```diff
@@ -1,30 +1,60 @@
 /// <reference types="vitest" />
 import react from '@vitejs/plugin-react-swc';
+import { ClientRequest } from 'node:http';
 import { fileURLToPath } from 'node:url';
-import { defineConfig } from 'vite';
+import { defineConfig, HttpProxy, loadEnv } from 'vite';
+
+const target = 'https://jodkehitys.fi';
+const cmsConfigure = (env: Record<string, string>) => (proxy: HttpProxy.Server) => {
+  proxy.on('proxyReq', (proxyReq: ClientRequest) => {
+    proxyReq.setHeader('Cookie', `JODTOKEN=${env.JODTOKEN}`);
+    proxyReq.setHeader('Accept', 'application/json');
+    proxyReq.setHeader('Authorization', 'Basic ' + Buffer.from(`${env.CMSUSER}:${env.CMSPASSWORD}`).toString('base64'));
+  });
+};
```

Comment:
Is it better to define this like this rather than directly in the proxy headers section?